best_practices.rst

```diff
@@ -0,0 +1,474 @@
+The Symfony Framework Best Practices
+====================================
+
+This article describes the **best practices for developing web applications with
+Symfony** that fit the philosophy envisioned by the original Symfony creators.
+
+If you don't agree with some of these recommendations, they might be a good
+**starting point** that you can then **extend and fit to your specific needs**.
+You can even ignore them completely and continue using your own best practices
+and methodologies. Symfony is flexible enough to adapt to your needs.
+
+This article assumes that you already have experience developing Symfony
+applications. If you don't, read first the rest of the `Symfony documentation`_.
+
+.. tip::
+
+    Symfony provides a sample application called `Symfony Demo`_ that follows
+    all these best practices, so you can experience them in practice.
+
+Creating the Project
+--------------------
+
+Use the Symfony Binary to Create Symfony Applications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Symfony binary is an executable command created in your machine when you
+`download Symfony`_. It provides multiple utilities, including the simplest way
+to create new Symfony applications:
+
+.. code-block:: terminal
+
+    $ symfony new my_project_name
+
+Under the hood, this Symfony binary command executes the needed `Composer`_
+command to create a new Symfony application based on the current stable version.
+
+Use the Symfony Skeleton to Create New Symfony-based Projects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Symfony provides two "skeletons" (or "project templates") to
+:ref:`create new Symfony projects <creating-symfony-applications>`. Use the
+minimal "skeleton" to start small and get the best performance and grow your
+application later as needed.
```

Comment:
Do we still need this one? As we recommend using the CLI, it will use this default/minimal skeleton automatically.